packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -187,6 +207,8 @@ export class DataVisualizerGraph
 	 * {@link DataVisualizerGraph.rootData | root shared objects}.
 	 */
 	public async renderRootHandles(): Promise<Record<string, RootHandleNode>> {
+		// TODO: We should be rendering the DataObject from the root, but this requires change in the devtools package in general.
```

Comment:
I don't think this is true. The calling code should be able to just specify something like:
```typescript
{ rootDataObject: rootDataObject.handle }
```
and your code should work